cofmpy/data_stream_handler/csv_data_stream_handler.py

```diff
@@ -68,6 +69,33 @@ def get_data(self, t: float):
             t (float): timestamp to get the data.
 
         Returns:
-            float: data value at the requested time.
+            dict: for the requested time, returns dict of values associated to endpoints
+                under the data handler scope (see BaseDataStreamHandler.is_equivalent_stream).
+                Format : {(fmu_i, var_j): value_1, (fmu_k, var_l): value_2}, ...}
+        """
+        out_dict = {}
+        for (node, endpoint), stream_alias in self.alias_mapping.items():
+            out_dict[(node, endpoint)] = self.interpolator(
+                self.data["t"], self.data[stream_alias], [t]
+            )[0]
+
+        return out_dict
+
+    # pylint: disable=W0237
+    def is_equivalent_stream(self, path, interpolation="previous") -> bool:
```

Comment:
In the base class, `is_equivalent_stream` takes a config dict as argument. Is it possible to use this same signature here?

```python
def is_equivalent_stream(self, config) -> bool:
    return config["path"] == self.path
```

Reply:
I might miss something, but in the Coordinator, we could call `dh.is_equivalent_stream(config["config"])` instead of `dh.is_equivalent_stream(**config["config"])`. There is no need to unpack the arguments. If child methods accept a dict instead of unpacked arguments, we could give a config dict to `is_equivalent_stream`, isn't it? With no changes to the handler instantiation and to the config format in the ConfigParser